_posts/2016-04-05-team-size-and-why-it-matters.md

```diff
@@ -0,0 +1,56 @@
+
+A small team enables people to be part of a team where everyone can communicate efficiently, helping to build a team which is trusting and motivated.
+
+### 2\. Management
+Have you ever been in a planning meeting with 10+ people? Or a retrospective with even more? It becomes significantly harder to coordinate a meeting with a large number of people, and often it also takes significantly longer to achieve the same thing. Large teams generate a lot of complexity which can be difficult to manage with process. Making sure everyone understands the current process’ policies, how they have been defined and importantly also in agreement to following them. 
+
+These effects can cause frustration and lack of belief in the process, where some ceremonies can feel too challenging to organise, or take too much time away from actual development.
+
+Smaller teams help members stay in control of their ceremonies and process, making it easier to manage, and allowing the focus to be on development, improvement and working software.
+
+### 3\. Engagement
+It’s important for a person to feel a part of the team they work in. Big teams make it more difficult for people to communicate effectively, to build trust, and to have a significant voice in discussions. This can lead to it being more difficult to be an engaged member. *[Social loafing](https://en.wikipedia.org/wiki/Social_loafing)* is a phenomenon that also helps to describe a disadvantage to larger teams, which is where people exert less effort when working with a greater number of people as it feels less important for them to do so.
+
+In a smaller team, people can feel like they have a much more significant part to play. People have more chance to have a say in how or what things are done.
+
+### 4\. Team Knowledge
+In larger teams, silos of knowledge are much more likely to occur. This can be caused by multiple projects being worked on at the same time. Even if people are free to pick up any work in the backlog, they are likely not to switch between different work streams after they start. This is to avoid context switching, and the want to finish a project once started. Secondly, it can occur due to the range of skills in the team. If someone has experience with database design, they will likely want to or be expected to pick up the database related work. Although this allows people to specialise in areas, it also reinforces silos of knowledge and skills in the team. The easiest way to think about why this isn’t a great idea is the bus factor. This describes the number of team members that can be lost from a team (ie. hit by a bus) before it is unable to continue working. 
```

Comment:
"and the desire" is exactly what I was trying to go for! Thanks

On Wed, Apr 6, 2016 at 9:11 AM medains <notifications@github.com> wrote:

> In _posts/2016-04-05-team-size-and-why-it-matters.md
> <https://github.com/skybet/skybet.github.io/pull/82#discussion_r58666618>:
>
> > +A small team enables people to be part of a team where everyone can communicate efficiently, helping to build a team which is trusting and motivated.
> > +
> > +### 2\. Management
> > +Have you ever been in a planning meeting with 10+ people? Or a retrospective with even more? It becomes significantly harder to coordinate a meeting with a large number of people, and often it also takes significantly longer to achieve the same thing. Large teams generate a lot of complexity which can be difficult to manage with process. Making sure everyone understands the current process’ policies, how they have been defined and importantly also in agreement to following them.
> > +
> > +These effects can cause frustration and lack of belief in the process, where some ceremonies can feel too challenging to organise, or take too much time away from actual development.
> > +
> > +Smaller teams help members stay in control of their ceremonies and process, making it easier to manage, and allowing the focus to be on development, improvement and working software.
> > +
> > +### 3\. Engagement
> > +It’s important for a person to feel a part of the team they work in. Big teams make it more difficult for people to communicate effectively, to build trust, and to have a significant voice in discussions. This can lead to it being more difficult to be an engaged member. *[Social loafing](https://en.wikipedia.org/wiki/Social_loafing)* is a phenomenon that also helps to describe a disadvantage to larger teams, which is where people exert less effort when working with a greater number of people as it feels less important for them to do so.
> > +
> > +In a smaller team, people can feel like they have a much more significant part to play. People have more chance to have a say in how or what things are done.
> > +
> > +### 4\. Team Knowledge
> > +In larger teams, silos of knowledge are much more likely to occur. This can be caused by multiple projects being worked on at the same time. Even if people are free to pick up any work in the backlog, they are likely not to switch between different work streams after they start. This is to avoid context switching, and the want to finish a project once started. Secondly, it can occur due to the range of skills in the team. If someone has experience with database design, they will likely want to or be expected to pick up the database related work. Although this allows people to specialise in areas, it also reinforces silos of knowledge and skills in the team. The easiest way to think about why this isn’t a great idea is the bus factor. This describes the number of team members that can be lost from a team (ie. hit by a bus) before it is unable to continue working.
>
> "and the want" reads poorly... it should be either "and they want"
> referring back to the previous sentence's "they" or a more general "and the
> desire"
>
> —
> You are receiving this because you authored the thread.
>
> Reply to this email directly or view it on GitHub
> <https://github.com/skybet/skybet.github.io/pull/82/files/f91c5ff56380251318eb9e558862530404ced272#r58666618>
>